aws-cf-template.json

```diff
@@ -62,6 +62,29 @@
                 }
               ]
             }
+          },
+          {
+            "PolicyName": "STSFederationToken",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Action": [
+                    "sts:GetFederationToken"
+                  ],
+                  "Effect": "Allow",
+                  "Resource": [
```

Comment:
Ten `Resource` tady můžeš nastavit na `*`. Ty práva se berou z usera který to volá https://docs.aws.amazon.com/STS/latest/APIReference/API_GetFederationToken.html